docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,115 @@
+
+   * Deploy a load balancer with [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+#### Use an Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`, use the URL:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) document.
+
+{{% notice warning %}}
+Exposing administrative, RMI, or T3 capable channels using a Kubernetes `NodePort`
+can create an insecure configuration. In general, only HTTP protocols should be made available externally and this exposure
+is usually accomplished by setting up an external load balancer that can access internal (non-`NodePort`) services.
+For more information, see [T3 channels]({{<relref "/security/domain-security/weblogic-channels#weblogic-t3-channels">}}).
+{{% /notice %}}
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For information about ingresses, see the [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer:
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+1. For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server, supply a URL that resolves to the load balancer host and ingress that you supplied in the previous step. For example:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/
+   ```
+   Where:
+
+     * `${HOSTNAME}` is where the ingress load balancer is running, in this case, the `External-IP` address of the Traefik load balancer.
+
+     * To determine the `${LB_PORT}` when using a Traefik load balancer:
+
+        `$ export LB_PORT=$(kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.ports[?(@.name=="web")].nodePort}')`
+
+### Test
+
+To verify that your WebLogic Server Administration Server URL is correct, and to verify that that your load balancer
+or `NodePort` are working as expected, run the following curl commands at the same location as your browser:
+
+
+```
+$ curl --user username:password \
+     http://${HOSTNAME}:${LB_PORT}/management/weblogic/latest/domainRuntime?fields=name\&links=none ; echo
+
+$ curl --user username:password \
```

Comment:
This seems to be a duplicate of the curl command

Reply:
One curl command queries the domainRuntime and the other, the serverRuntime.

Reply:
ok